pkgs/development/python-modules/hf-xet/default.nix

```diff
@@ -28,7 +28,7 @@ buildPythonPackage rec {
       src
       sourceRoot
       ;
-    hash = "sha256-D6R2FFGDKB4VgMkflF441Ki8o1RCwBoumQ4oeNL/fnc=";
+    hash = "sha256-qZUkHhE2lE+wNQkOPNuMoCUHcilwzRd0UG5JHC0vb+M=";
   };
 
```

Comment:
`meta.homepage` leads to a 404; did you mean https://github.com/huggingface/xet-core/tree/main/hf_xet?

Reply:
Good catch, fixed!